sql/core/src/test/java/test/org/apache/spark/sql/JavaBeanDeserializationSuite.java

```diff
@@ -590,9 +590,9 @@ public Item call(Item item1, Item item2) throws Exception {
             .reduceGroups(rf);
 
     List<Tuple2<String, Item>> expectedRecords = Arrays.asList(
-            new Tuple2("a", new Item("a", 8)),
-            new Tuple2("b", new Item("b", 3)),
-            new Tuple2("c", new Item("c", 2)));
+            new Tuple2<>("a", new Item("a", 8)),
```

Comment:
There should be many similar cases in the test code. Because they are test codes, I have not touched them before to avoid conflicts. If we need to fix them, I think we should fix all in one.

WDYT @HyukjinKwon